### PR TITLE
Fix/2255

### LINKED ISSUE #2255: The "Copy" icon and the "Press Ctrl-V to paste." text are missing from the "Link Copied" system notification
[Affected versions]:
- Nightly 54.0a1 (Build ID: 20170301030202)	
- Page Shot 5.2.4829

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- The "xpinstall.signatures.required" is set to "false" and latest Page Shot (5.2.4289) custom build is installed on a new clean profile.

[Steps to reproduce]:
1. Open Firefox with the profile from prerequisites and navigate to any website.
2. Click the Page Shot button from browser toolbar.
3. Perform a selection and save it.
4. Observe the "Link Copied" system notification.

[Expected result]:
- The "Copy" icon and the "Press Ctrl-V to paste." text are correctly displayed.

[Actual result]:
- The "Copy" icon and the "Press Ctrl-V to paste." text are no longer displayed.

[Notes:]
- Here are two screen shots with the issue:
- Mac OS:
mac os system notification

- Windows:
windows page shot system notification

### PR DESCRIPTION
**Pull request by niharikak101**
fixes #2255

### CODE REVIEW

File: webextension/background/takeshot.js [hunk 1]
```diff
@@ -35,9 +37,10 @@ window.takeshot = (function () {
       clipboard.copy(shot.viewUrl);
       chrome.notifications.create(id, {
         type: "basic",
-        iconUrl: "img/clipboard-32.png",
+        iconUrl: "../icons/clipboard-32.png",
         title: "Link Copied",
-        message: "The link to your shot has been copied to the clipboard"
+        message: "The link to your shot has been copied to the clipboard. Press "
+        + pasteSymbol + "-V to paste."
```
Comment (ianb) [inline webextension/background/takeshot.js]: We also use ES6 here, so you can do:

    `The link to your shot has been copied to the clipboard. Press ${pasteSymbol}-V to paste.`

but this is fine too

**Comment by ianb** [comment]
Thanks, and good that you caught the bad icon URL too!

**Comment by niharikak101** [comment]
@ianb Thank you! I Will definitely keep that in mind for next time

**Comment by ianb** [comment]
Another note: I noticed the code had some trailing whitespace.  If you can configure your editor to strip trailing whitespace on each line, that helps avoid unnecessary diffs when the code is later edited.

**Comment by niharikak101** [comment]
@ianb I'll do that! Thanks

**Pull request merged**